### ISSUE TITLE
(WCAG 2.1) Language selection is missing `lang` attribute on foreign language options

### ISSUE THREAD

**Issue by jeanniefu** [civiform/civiform#3546]
Required to meet WCAG 2.1 guidelines. Identified during an accessibility audit against WCAG 2.1.

**WCAG SC**:  [3.1.2. Language of Parts](https://www.w3.org/TR/WCAG21/#language-of-parts)

**Observed results**: The foreign language options displayed do not have lang set programmatically inline. This causes screen reader to mis-pronounce the text.

**How to fix**: Add `lang` attribute to radio buttons on language selection page. AND add to the dropdown menu in the header.

(Jeannie's note): We may also want to add this throughout the CiviForm website to all translated text. Need to check with a11y team, since we didn't get any specific instructions about the rest of the site.

Example:
```
<p>And with a certain <span lang=""fr"">je ne sais quoi</span>,
   she entered both the room, and his life, forever. <q>My name
   is Natasha,</q> she said. <q lang=""it"">Piacere,</q>
   he replied in impeccable Italian, locking the door.
</p>
```

**Comment by clouser2** [comment]
Adding a `lang` attribute to the radio buttons definitely makes sense on the Language selection page since it is primarily rendered in English with a bunch of different language choices.

However, after making this selection, the rest of the UI should be fully localized after that. Would we need to add a `lang` tag to each element in the UI in that case or would we be able to set it at a much higher-level (e.g. `<body>` tag) for situations where we're consistently displaying in the applicant's preferred language?

**Comment by clouser2** [comment]
It appears that we set the `lang` attribute on the `<html>` element after a language has been chosen.

**Comment by jeanniefu** [comment]
Good point, I'll leave the scope of this bug to just be for the language selection page. I edited the text at the top of this bug. I'm actually not sure what the right behavior is when the entire website is translated. We didn't get any specific feedback regarding that (since I didn't ask them to test it).

**Comment by jeanniefu** [comment]
Amazing thank you!

Also updated the initial comment. I forgot to mention this needs to be updated in two places: (1) on the language selection page and (2) on the dropdown menu in the header once you start applying.
